ui/src/app/views/concept-homepage/component.ts

```diff
@@ -46,13 +54,23 @@ export class ConceptHomepageComponent implements OnInit {
   conceptAddModal: ConceptAddModalComponent;
 
   conceptDomainList: Array<DomainInfo> = [];
+  conceptDomainCounts: Array<DomainCount> = [];
 
   concepts: Array<Concept> = [];
   conceptsCache: Array<ConceptCacheSet> = [];
 
+  completedDomainSearches: Array<Domain> = [];
+
+  vocabularies: Array<VocabularyCountSelected> = [];
+
   wsNamespace: string;
   wsId: string;
 
+  // For some reason clr checkboxes trigger click events twice on click. This
+  // is a workaround to not allow multiple filter events to get triggered.
+  blockMultipleSearchFromFilter = true;
```

Comment:
Have you tried `e.stopPropagation()` in your click callback??

Reply:
Will try. Let me look into that.

Reply:
Tried it, it blocked changing the values too. I think Clarity needs the propagation, but for some reason that triggers multiple events... :(